browser.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
+const https = require('https');
+const fs = require('fs');
 const electron = require('electron');
+const { dialog } = require('electron').remote;
```

Comment:
Please use the already defined ```electron``` variable from the line above *(line 4)*, or call it directly from ```electron.remote```